staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -1924,6 +1924,28 @@ Rules:
 Annotations:   <none>
 Events:        <none>` + "\n",
 		},
+		"DefaultBackend": {
+			input: fake.NewSimpleClientset(&networkingv1.Ingress{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "bar",
+					Namespace: "foo",
+				},
+				Spec: networkingv1.IngressSpec{
+					DefaultBackend: &backendV1,
+				},
+			}),
+			output: `Name:             bar
+Namespace:        foo
+Address:
```

Comment:
Apparently it's missing 10 spaces here, like: https://github.com/kubernetes/kubernetes/blob/b895f8c7a697a97acf28067b2f38a0dc7d822c02/staging/src/k8s.io/kubectl/pkg/describe/describe_test.go#L1770

Reply:
Right, good catch.